app/models/user.rb

```diff
@@ -1,8 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- You have done well so far, but according to the project requirements, please make sure your database is confirmable by adding the `:confirmeable` module to your user model